src/include/sof/math/power.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+#ifndef __POWER_SCALAR_FUNCTION_H__
+#define __POWER_SCALAR_FUNCTION_H__
+
+/* Include Files */
+#include <stdint.h>
+/* define constant */
+#define B_NUMERATOR (0x4000000000000ULL)
+#define MAX_LIMIT 0x8000
```

Comment:
The #define macros for number constants should be in some private header file and not brought to application component context if they are for internals of functions only. Or at least prefix the constants like done above for log header.

Reply:
Sure , I will add prefix like log header to power constant